SMRA/Simple SMR01 extract.R

```diff
@@ -38,3 +38,6 @@ smr01_query
 
 # Return the data
 smr01_extract <- collect(smr01_query)
+
+# close the connection
+dbDisconnect(smra_conn)
```

Comment:
If we've closed the connection, we may as well remove it too.
```suggestion
# close and remove the connection
dbDisconnect(smra_conn)
rm(smra_conn)
```

Reply:
I like it James - makes perfect sense